### DISCUSSION TITLE
My awakened poe trade broken

### DISCUSSION THREAD

**Discussion by etterna0** [SnosMe/awakened-poe-trade#1462] [Q&A] [answered]
I press CTR+D or SHIFT+SPACE and it doesn't work.

I have reinstalled the application 50 times already. both through the installer, and just an exe file, and even an archive. running “as administrator” doesn't help either. the game is in a window. other similar programs work, but not APT.

**Comment by jbrocato** [comment]
It used to work.  I have been trying to debut it myself since end of July.  The same is happening for me as well.    Checked %appdata% for any information.  I also used the uninstall, however it is not a clean uninstall.

I am going to delete the folder in AppData\Roaming since even after the uninstall there is the log and manifest from May of 2023, when it was first installed and see if that helps.  Also will trying and search the registry for anything lingering.  Then try yet another reinstall.  

Any suggestions to troubleshoot would be much appreciated.

2024/09/03-19:14:44.657 7e34 Reusing MANIFEST C:\Users\xxxxxx\AppData\Roaming\awakened-poe-trade\Local Storage\leveldb/MANIFEST-000001
2024/09/03-19:14:44.659 7e34 Recovering log #3
2024/09/03-19:14:44.659 7e34 Reusing old log C:\Users\xxxxx\AppData\Roaming\awakened-poe-trade\Local Storage\leveldb/000003.log

**Reply by jbrocato** [reply, marked as answer]
So if "for all users" is chosen during installation there is an application crash event viewer error.  If  "current user only" is chosen the install works fine.  

It is now working again.  Maybe there was a poe crash and somehow awakened appdata file got corrupted...who knows.  

to fix, i deleted the AppData\Roaming\awakened-poe-trade folder.  Installed awakened poe trade for "current user only".

maybe there is a issue with uses that installed for "all users" with the latest 3.25 exe.

**Reply by etterna0** [reply]
thank you, it really works!
